documentation/design_docs/README.adoc

```diff
@@ -3,29 +3,29 @@ EnMasse
 -------
 
 Documents relating to EnMasse project. For setting up EnMasse, see
-link:getting-started/README.adoc[Getting Started]. To setup EnMasse with
+link:design_docs/getting-started/README.adoc[Getting Started]. To setup EnMasse with
```

Comment:
I don't think these links should have the design_docs prefix, as the doc they are in is itself in that design_docs folder. It is the toplevel README, i.e. directly under enmasse, that needs its links updated.

Reply:
@grs, got it, thanks, Gordon! Will revert this one and fix the top-level README file.